src/openrct2/ride/RideColour.h

```diff
@@ -13,7 +13,24 @@
 
 struct TrackColour
 {
-    uint8_t main;
-    uint8_t additional;
-    uint8_t supports;
+    colour_t main;
```

Comment:
Needs a `common.h` header import. (Though we should probably move the type definition over to `Colour.h`.)

Reply:
Done. Could you create an issue for the latter?